### DISCUSSION TITLE
Deterministic USB Mapping

### DISCUSSION THREAD

**Discussion by mdlarussa** [aldostools/webMAN-MOD#800] [Q&A] [answered]
Is there was a way to force map a drive to a particular USB port, or force EXTDG to use a specific drive?  Running with three external drives.  One is nearly empty, the other two are maxed out minus 5-10 GB.  when I run /extgd.ps3?1 in my autoexec.bat, it is utilizing one of the full drives.  not the one with 1.5 TB free.  Do I shuffle around which ports each drive is connected to?  Is there something like a .cfg I could put on each drive to say this should be USB006 and the one to use for EXTDG?  I assume there's no way around having whatever is plugged into USB2 to be USB000, but does it have to be a random grab for the higher drives?  I had this working once with the three drives and EXTDG pointed game to USB006/GAMEI, which was perfect.  Took the drives off and modified them on windows, plugged them back in, and no go.  I guess I'll have to mark which drive goes on which port, if that's the true fix.

**Comment by aldostools** [comment, marked as answer]
/extgd.ps3 uses the first USB drive with FAT32 found.
it first checks the first drive with the folder GAMEI already created.
if a GAMEI folder is not found, it creates the folder in the first USB drive found.

If you want to force to use one USB drive, you must rename or delete the GAMEI folder in the other drives.
Or use the web command http://0/remap.ps3/dev_hdd0/game&to=/dev_usb006/GAMEI
